backend/src/test/java/de/sambam/gamenighttracker/controller/UserControllerTest.java

```diff
@@ -71,21 +78,27 @@ public void setup() {
                 .build();
 
         userDb.deleteAll();
-        userDb.save(User.builder()
-                .id("1")
-                .userName("Sanne")
-                .playedGames(List.of(game1, game2, game3))
-                .build()
-        );
+        appUserDb.deleteAll();
+        userDb.save((User.builder()
+                .userName("sanne")
+                .playedGames(Arrays.asList(game1, game2, game3))
+                .build()));
     }
 
 
     @Test
     @DisplayName("GET request to /user/games should return all the games from the Db")
     public void getGameListTest() {
+        //GIVEN
+
         //WHEN
-        ResponseEntity<Game[]> getResponse = testRestTemplate.getForEntity(
-                "http://localhost:" + port + "/user/games", Game[].class);
+        String jwtToken = logintoApp();
+        HttpHeaders header = new HttpHeaders();
+        header.setBearerAuth(jwtToken);
+        HttpEntity<Void> entity = new HttpEntity<>(header);
+
+        ResponseEntity<Game[]> getResponse = testRestTemplate.exchange(
+                "http://localhost:" + port + "/user/games", HttpMethod.GET, entity, Game[].class);
```

Comment:
Think of a factory for all your `Game.builder().....build()` action on top. This would reduce the amount of boilerplate significatly.